android/tunnel/CMakeLists.txt

```diff
@@ -15,16 +15,16 @@ if(CCACHE_FOUND)
 endif(CCACHE_FOUND)
 
 # Work around https://github.com/android-ndk/ndk/issues/602
-set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fuse-ld=gold")
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fuse-ld=lld")
```

Comment:
Why a different linker?

Reply:
gold is depricated and was removed in the new ndk version we're using now